src/variables/components/TypeAheadVariableDropdown.tsx

```diff
@@ -208,18 +304,23 @@ class TypeAheadVariableDropdown extends PureComponent<Props, MyState> {
             onCollapse={onCollapse}
             theme={DropdownMenuTheme.Amethyst}
           >
-            {shownValues.map(val => {
+            {shownValues.map(function(value, index) {
```

Comment:
```suggestion
            {shownValues.map((value, index) => {
```

should autobind for you and fix everything up, right?